### PR TITLE
Psr 457

### PR DESCRIPTION
**Pull request by EricSiegNW**
Adds the new progress Bar w/ mask to break it up into 12 chunks
Updates Insight Panel
New puzzle piece/selected puzzle piece

### CODE REVIEW

**Comment by dephillipsmichael** [comment]
I'm glad to see the progress bar mask solution worked out!

I did notice that the quality is blurry on a 3x device:
Screen Shot 2021-10-17 at 1 54 40 PM

Then I checked the assets, and you missed 1x and 3x
Screen Shot 2021-10-03 at 1 55 33 PM

File: Psorcast/Psorcast/MeasureTabViewController.swift [hunk 1]
```diff
@@ -92,6 +95,7 @@ open class MeasureTabViewController: UIViewController, UICollectionViewDataSourc
     var currentStatus: String?
     
     let showInsightTaskId = "showInsight"
+    let totalStudyWeeks = 12
```
Comment (dephillipsmichael) [inline Psorcast/Psorcast/MeasureTabViewController.swift:98]: This probably belongs in `MasterScheduleManager`

Comment (EricSiegNW) [reply Psorcast/Psorcast/MeasureTabViewController.swift:98]: Waiting on assets for the other resolutions, but didn't think it made sense to wait.

**Review by DanWebster — CHANGES_REQUESTED** [review]
@EricSiegNW  I'm getting build errors with a missing CLLocationManager, is there an update to SageResearch I need to get from Carthage? 
Screen Shot 2021-10-04 at 7 31 56 AM

**Comment by EricSiegNW** [comment]
Hey Dan,

It did require a change to SageResearch, which has been merged and pointed at. You might need to run update in Carthage to get to that?

**Pull request merged**